Resources/views/backend/swag_backend_order/view/main/customer_information/shipping.js

```diff
@@ -37,8 +37,13 @@ Ext.define('Shopware.apps.SwagBackendOrder.view.main.CustomerInformation.Shippin
         me.customerStore = me.subApplication.getStore('Customer');
         me.customerStore.on('load', function () {
             if (Ext.isObject(me.customerStore) && me.customerStore.count() == 1) {
-                me.shippingStore = me.customerStore.getAt(0).shipping();
+                var customerModel = me.customerStore.getAt(0);
+                me.shippingStore = customerModel.shipping();
                 me.shippingAddressComboBox.bindStore(me.shippingStore);
+
+                if (customerModel.getDefaultBilling().get('id') !== customerModel.getDefaultShipping().get('id')) {
```

Comment:
This could result into a JS error, cause you leftJoin

Reply:
@shyim I added a check if both stores are populated and therefore can be used.